actions/workflows/bwc_prep_dev.yaml

```diff
@@ -1,124 +1,153 @@
-        #         hosts: <% $.host %>
-        #         cwd: <% $.cwd %>
-        #     on-success:
-        #         - prep_bwc_topology
-        # prep_bwc_topology:
-        #     action: st2cd.bwc_prep_dev_for_bwc_topology
-        #     input:
-        #         version: <% $.dev_version %>
-        #         fork: <% $.fork %>
-        #         local_repo: <% 'bwc_topology_' + $.local_repo_sfx %>
-        #         hosts: <% $.host %>
-        #         cwd: <% $.cwd %>
-        #     on-success:
-        #         - prep_bwc_cli
-        # prep_bwc_cli:
-        #     action: st2cd.bwc_prep_dev_for_bwc_cli
-        #     input:
-        #         version: <% $.dev_version %>
-        #         fork: <% $.fork %>
-        #         local_repo: <% 'bwc_cli_' + $.local_repo_sfx %>
-        #         hosts: <% $.host %>
-        #         cwd: <% $.cwd %>
-
+      cmd: echo `date +'%s'`_`awk -v min=100 -v max=999 'BEGIN{srand(); print int(min+rand()*(max-min+1))}'`
+    next:
+      - when: <% succeeded() and (ctx().host = null) %>
+        publish:
+          - local_repo_sfx: <% result().stdout %>
+        do:
+          - get_host
+      - when: <% succeeded() and (ctx().host != null) %>
+        publish:
+          - local_repo_sfx: <% result().stdout %>
+        do:
+          - prep
+  get_host:
+    action: linux.dig
+    input:
+      hostname: st2-build-slave-ubuntu.service.consul
+      rand: true
+      count: 1
+    next:
+      - when: <% succeeded() %>
+        publish:
+          - host: <% result().result[0] %>
+        do:
+          - prep
+  prep:
+    next:
+      - do:
+          - prep_bwc_enterprise_pkg
+  prep_bwc_enterprise_pkg:
+    action: st2cd.bwc_prep_dev_for_enterprise_pkg
+    input:
+      project: bwc-enterprise-package
+      version: <% ctx().version %>
+      fork: <% ctx().fork %>
+      local_repo: <% 'bwc_enterprise_package_' + ctx().local_repo_sfx %>
+      hosts: <% ctx().host %>
+      cwd: <% ctx().cwd %>
+    next:
+      - when: <% succeeded() %>
+        do:
+          - prep_st2enterprise_ldap
+  prep_st2enterprise_ldap:
+    action: st2cd.st2_prep_dev_for_enterprise_ldap
+    input:
+      project: st2-enterprise-auth-backend-ldap
+      version: <% ctx().dev_version %>
+      fork: <% ctx().fork %>
+      local_repo: <% 'st2enterprise_auth_ldap_' + ctx().local_repo_sfx %>
+      hosts: <% ctx().host %>
+      cwd: <% ctx().cwd %>
+    next:
+      - when: <% succeeded() %>
+        do:
+          - prep_bwc_ui
+  prep_bwc_ui:
+    action: st2cd.bwc_prep_dev_for_bwc_ui
+    input:
+      project: bwc-ui
+      version: <% ctx().dev_version %>
+      fork: <% ctx().fork %>
+      local_repo: <% 'bwc_ui_' + ctx().local_repo_sfx %>
+      hosts: <% ctx().host %>
+      cwd: <% ctx().cwd %>
+    next:
+      # Commented out in the original workflow
+      # - when: <% succeeded() %>
+      #   do:
+      #     - prep_bwc_ipfabric_suite
```

Comment:
Let's just remove commented out code since we're touching this.

Reply:
Fixed in 8bfeff9.